### PR TITLE
Features/views

### PR DESCRIPTION
**Pull request by harveyj123**
css

### CODE REVIEW

File: templates/login/instructions/nodefault.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="innerWidth2 bodyText">
+<div class="errPage">
```
Comment (abitblue) [inline templates/login/instructions/nodefault.html:1]: Bad name, we have more dedicated /error page.

**Review by abitblue — COMMENTED** [review]
Logs should not get committed.

Instruction buttons do not work. I need to click the text, but clicking on the button doesn't work.
Inconsistent button styling between pages, and they feel clunky without some form of animation.

Border ratios between sides do not feel correct. Lots of text feel too large compared to other pieces of text.

**Comment by abitblue** [comment]
Suggestion: Clean up what we have now instead of trying to add flexboxes?
I don't think flexboxes are needed here. Firstly, IE support. While it is no longer supported, many legacy systems around the building still rely on it. We don't have a choice for those.
I also find that flexboxes are best used when things need to dynamically scale all the time, such as when there are panels and dropdowns and things need to move. In our case, we're not doing that. In our case, most of the site is very static, and lots of what is seen can be solved more simply with basic CSS, instead of adding `display: flex` to most objects.

I'd like to hear what @IvanZ505 has to say: if he agrees or not.

**Comment by IvanZ505** [comment]
> 

Yeah I agree with this. The issue is we have so many files that don't follow the "standard" HTML style but is instead generated or extended through the use of Django. The issue we had last year with adding a ton of flexbox was it was extremely hard to mesh together everything while keeping every single site looking the way we want it to look.

**Comment by harveyj123** [comment]
I think i just ditch it and go with the original css and just center some things like header when it gets smaller

**Comment by abitblue** [comment]
Closing PR.

**Comment by abitblue** [comment]
Addresses Issue #8.

**Pull request closed without merging**